generator/src/main/java/org/corfudb/generator/LongevityApp.java

```diff
@@ -52,7 +55,16 @@ public LongevityApp(long durationMs, int numberThreads, String configurationStri
         this.numberThreads = numberThreads;
 
         operationQueue = new ArrayBlockingQueue<>(QUEUE_CAPACITY);
-        rt = new CorfuRuntime(configurationString).connect();
+
+        rt = new CorfuRuntime(configurationString);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "CorfuRuntime"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)